utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -16,3 +16,10 @@ annotation class SplashScreenWelcomeMsg
 val SPLASH_SCREEN_WELCOME_MSG = "splash_screen_welcome_msg"
 val SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE = false
 val SPLASH_SCREEN_WELCOME_MSG_VALUE = true
+
+@Qualifier
+annotation class MyDownloads
+
+val MY_DOWNLOADS_FLAG = "my_downloads_flag"
+val MY_DOWNLOADS_IS_ENABLED_DEFAULT = true
```

Comment:
This should be default false--we don't want to change the developer experience since this workaround is really impactful (& will probably break tests).

Reply:
fixed